radware-alteon/info.json

```diff
@@ -0,0 +1,246 @@
+{
+    "name": "radwareAlteon",
+    "version": "1.0.8",
+    "label": "Radware Alteon",
+    "description": "",
+    "publisher": "Danny",
+    "icon_small_name": "RDWR logo small.png",
+    "icon_large_name": "RDWR logo large.png",
+    "category": "",
+    "configuration": {
+        "fields": [
+            {
+                "title": "URL",
+                "type": "text",
+                "name": "uRL",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "https://192.168.1.1",
+                "tooltip": "the api url for the Radware Alteon"
+            },
+            {
+                "title": "b64EncodedAuth",
+                "type": "password",
+                "name": "b64EncodedAuth",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": null
+            },
+            {
+                "title": "Username",
+                "type": "text",
+                "name": "username",
+                "required": false,
+                "visible": false,
+                "editable": true
+            },
+            {
+                "title": "Password",
+                "type": "password",
+                "name": "password",
+                "required": false,
+                "visible": false,
+                "editable": true
+            },
+            {
+                "title": "UserPassEncoding",
+                "type": "text",
+                "name": "userPassEncoding",
+                "required": true,
```

Comment:
```suggestion
                "required": false,
```

Reply:
user pass encoding is mandatory so not changing this.